packages/controller-utils/src/types.ts

```diff
@@ -27,6 +27,13 @@ export enum NetworksChainId {
   rpc = '',
 }
 
+export const NetworkIds = {
```

Comment:
Nit: We should probably call this `NetworkId` if we convert it to an `enum` (as I've suggested in another comment as a way to fix the build error)